packages/asc-ui/src/components/Button/ButtonStyle.ts

```diff
@@ -29,6 +32,16 @@ const ButtonStyle = styled(ButtonBaseStyle)<Props>`
       `}
   }
 
+  /* stylelint-disable-next-line selector-type-no-unknown */
+  & > ${IconStyle}:first-child {
+    margin-right: 5px;
+  }
+
+  /* stylelint-disable-next-line selector-type-no-unknown */
+  & > ${IconStyle}:last-child {
+    margin-left: 5px;
+  }
+
   & > svg {
```

Comment:
perhaps this can be removed? because we use the svg already inside an icon? perhaps pass the icon like a prop, and then wrap it inside an <Icon>, that way we are ensured the button's icon(s) is / are always wrapped inside an <Icon>

Reply:
removed